### PR TITLE
webdavclient2 0.1.1

### PR DESCRIPTION
**Pull request by ReimarBauer**

### CODE REVIEW

**Comment by ReimarBauer** [comment]
Hi @dougalsutherland 
have I here the same problem as with the green build with appveyor?

File: recipes/webdavclient2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
```
Comment (dougalsutherland) [inline recipes/webdavclient2/meta.yaml]: Please delete this comment...but actually, looking at the package, you could uncomment the `noarch: python` line.

File: recipes/webdavclient2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - requests
+    - lxml
+    - libxml2
```
Comment (dougalsutherland) [inline recipes/webdavclient2/meta.yaml]: I don't think the package explicitly needs `libxml2` and `libxslt`, just a working `lxml` install.

File: recipes/webdavclient2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - libxslt
+
+test:
+  imports:
+    - webdav2
+    - webdav2.client
+
```
Comment (dougalsutherland) [inline recipes/webdavclient2/meta.yaml]: There's a `wdc` script that you should test; just
```yaml
  commands:
    - wdc --help
```
 is probably fine.

File: recipes/webdavclient2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    webdavclient2
+    ============
+
+    Based on https://github.com/designerror/webdav-client-python
+    But uses `requests` instead of `PyCURL`
+  dev_url: https://github.com/appetito/webdav-client-python-2
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (dougalsutherland) [inline recipes/webdavclient2/meta.yaml]: Please delete this comment too.

File: recipes/webdavclient2/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+   but uses requests instead of PyCURL'
+
+  # The remaining entries in this section are optional, but recommended
+  description: |
+    WebDAV client, based on original package https://github.com/designerror/webdav-client-python but uses requests instead of PyCURL
+    webdavclient2
+    ============
+
+    Based on https://github.com/designerror/webdav-client-python
+    But uses `requests` instead of `PyCURL`
```
Comment (dougalsutherland) [inline recipes/webdavclient2/meta.yaml]: You could clean this up a bit.

**Comment by ReimarBauer** [comment]
cool appveyor works :)

**Comment by ReimarBauer** [comment]
after this is on conda-forge I  push a recipe for fs.webdavfs ;)

File: recipes/webdavclient2/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    - setuptools
+  run:
+    - python
+    - requests
+    - lxml
+    - argcomplete
+
+test:
+  commands:
+    # script currently broken, issue #4
```
Comment (jakirkham) [inline recipes/webdavclient2/meta.yaml]: Should we comment `commands` then also as it is empty? Also do you mind adding the full link to the issue for clarity?

File: recipes/webdavclient2/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    # - wdc --help
+  imports:
+    - webdav2
+    - webdav2.client
+
+about:
+  home: https://github.com/appetito/webdav-client-python-2
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (jakirkham) [inline recipes/webdavclient2/meta.yaml]: Please ask upstream to add the license file to `MANIFEST.in` and please link that issue above in a comment. Thanks. 😄

File: recipes/webdavclient2/meta.yaml (hunk 6 above)
Comment (ReimarBauer) [reply recipes/webdavclient2/meta.yaml]: have not thought on just replying here. @dougalsutherland mentioned the command and by that I found the typo.

File: recipes/webdavclient2/meta.yaml (hunk 7 above)
Comment (jakirkham) [reply recipes/webdavclient2/meta.yaml]: xref: https://github.com/appetito/webdav-client-python-2/issues/6

File: recipes/webdavclient2/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+  run:
+    - python
+    - requests
+    - lxml
+    - argcomplete
+
+test:
+  # commands:
+    # script currently broken, issue https://github.com/appetito/webdav-client-python-2/issues/4
+    # - wdc --help
```
Comment (jakirkham) [inline recipes/webdavclient2/meta.yaml]: xref: https://github.com/appetito/webdav-client-python-2/issues/4

**Comment by jakirkham** [comment]
Thanks @ReimarBauer!

**Pull request merged**